tests/unit/src/main/scala/tests/debug/BaseBreakpointDapSuite.scala

```diff
@@ -116,6 +116,7 @@ abstract class BaseBreakpointDapSuite(
                 |  def main(args: Array[String]): Unit = {
                 |    this match {
                 |      case Bar() =>
+                |      case _ =>
```

Comment:
It seems that we are getting exhaustivity issues here, but it's not the point of this test.